sudoers.txt

```diff
@@ -0,0 +1,4 @@
+root ALL=(ALL) ALL
+user ALL=(ALL) NOPASSWD: ALL
```

Comment:
Allows `user` to use `sudo` without a password

Reply:
That's fine, but can we move this file (as well other docker specific files?) to a directory (e.g. `docker`).